### PR TITLE
chore: update eventpersister to store mau to mysql

### PR DESCRIPTION
**Pull request by masaaania**
- I updated the event persister to store MAU to MySQL.
- This app works without MySQL-related environment vars.
  - `event-persister-evaluation-event` and `event-persister-goal-event` don't have to set the env vars.
- If storing to MySQL fails, storing to kafka is not executed.

### CODE REVIEW

File: pkg/eventpersister/storage/v2/mau.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		created_at,
+		updated_at,
+		environment_namespace
+	) VALUES (
+		?, ?, ?, ?, ?, ?, ?
+	) ON DUPLICATE KEY UPDATE
+		event_count = event_count + 1,
+		updated_at = VALUES(updated_at)
+	`
+
```
Comment (cre8ivejp) [inline pkg/eventpersister/storage/v2/mau.go:56]: Please delete the empty line.

File: pkg/eventpersister/persister/persister.go [hunk 2]
```diff
@@ -248,6 +253,18 @@ func (p *Persister) send(messages map[string]*puller.Message) {
 					)
 				}
 			}
+
```
Comment (cre8ivejp) [inline pkg/eventpersister/persister/persister.go:256]: Please delete the empty line.

File: pkg/eventpersister/persister/persister.go [hunk 3]
```diff
@@ -248,6 +253,18 @@ func (p *Persister) send(messages map[string]*puller.Message) {
...
+				p.logger.Error(
+					"failed to store a mau",
+					zap.Error(err),
+					zap.String("id", id),
+					zap.String("environmentNamespace", environmentNamespace),
+				)
+				fails[id] = false
+				continue
+			}
+
```
Comment (cre8ivejp) [inline pkg/eventpersister/persister/persister.go:267]: Same here.

File: pkg/eventpersister/cmd/server/server.go [hunk 4]
```diff
@@ -223,6 +245,26 @@ func (s *server) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.L
 	return nil
 }
 
+func (s *server) createMySQLClient(
+	ctx context.Context,
+	registerer metrics.Registerer,
+	logger *zap.Logger,
+) (mysql.Client, error) {
+	if *s.mysqlUser == "" || *s.mysqlPass == "" || *s.mysqlHost == "" || *s.mysqlPort == 0 || *s.mysqlDbName == "" {
```
Comment (cre8ivejp) [inline pkg/eventpersister/cmd/server/server.go:253]: Is it possible for you to add a unit test to check those conditions?

Comment (masaaania) [reply pkg/eventpersister/cmd/server/server.go:253]: Let me add it in another PR 🙏

**Review by cre8ivejp — APPROVED** [review]
Nice, thanks!

**Pull request merged**